packages/agents-copilotstudio-client/src/connectionSettings.ts

```diff
@@ -3,38 +3,44 @@
  * Licensed under the MIT License.
  */
 
+import { AgentType } from './agentType'
+import { CopilotStudioConnectionSettings } from './copilotStudioConnectionSettings'
+import { PowerPlatformCloud } from './powerPlatformCloud'
+
 /**
  * Represents the settings required to establish a connection to Copilot Studio.
  */
-export class ConnectionSettings {
-  /** The client ID of the application. */
+export class ConnectionSettings implements CopilotStudioConnectionSettings {
   public appClientId: string = ''
-  /** The tenant ID of the application. */
   public tenantId: string = ''
-  /** The environment ID of the application. */
   public environmentId: string = ''
-  /** The cloud environment of the application. */
-  public cloud: string = ''
-  /** The custom Power Platform cloud URL, if any. */
+  public cloud?: PowerPlatformCloud
   public customPowerPlatformCloud?: string
-  /** The identifier of the agent. */
   public agentIdentifier?: string
-  /** The type of the Copilot agent. */
-  public copilotAgentType?: string
+  public copilotAgentType?: AgentType
+  public directConnectUrl?: string
+  public useExperimentalEndpoint?: boolean = false
```

Comment:
can you update the `env.TEMPLATE` in the sample to include these options? I'm missing useExperimentalEndPoint , DirectConnectionUrl and UseS2SConnection